index.html

```diff
@@ -2018,6 +2045,31 @@ <h2>Authentication</h2>
           </dd>
         </dl>
 
+        <pre class="example nohighlight" title="Authentication property
+                      containing three verification methods">
+{
+  "@context": "https://www.w3.org/ns/did/v1",
+  "id": "did:example:123456789abcdefghi",
+  <span class="comment">...</span>
+  "authentication": [
+    <span class="comment">// this method can be used to authenticate as did:...fghi</span>
+    "did:example:123456789abcdefghi#keys-1",
+    <span class="comment">// this method can be used to authenticate as did:...fghi</span>
+    "did:example:123456789abcdefghi#biometric-1",
```

Comment:
```suggestion
    <span class="comment">// this method can be used to authenticate as did:...fghi</span>
    "did:example:123456789abcdefghi#biometric-1",
```
I'm fairly certain the group would rail against the notion of embedding a biometric in the DID Document at this point... removing it from the example.